ci/pipelines/perf-tests.yml

```diff
@@ -619,6 +637,35 @@ jobs:
             AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
             AWS_SECRET_ACCESS_KEY: ((.:role.AWS_SECRET_ACCESS_KEY))
             AWS_SESSION_TOKEN: ((.:role.AWS_SESSION_TOKEN))
+          on_failure:
+            put: slack-notification
+            attempts: 10
+            params:
+              channel: '#govuk-pay-announce'
+              silent: true
+              text: ':red-circle: Performance test SelfServiceSimulation failed - <https://pay-cd.deploy.payments.service.gov.uk/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME|Concourse build #$BUILD_NAME>'
+              icon_emoji: ":concourse:"
+              username: pay-concourse
+    on_failure:
+      put: slack-notification
+      attempts: 10
```

Comment:
What does this `attempts` mean?

Reply:
It's just a retry mechanism for failure. So if the slack notification fails it will retry up to 10 times